src/Components/ui/SearchInput.tsx

```diff
@@ -93,7 +90,7 @@ const styles = StyleSheet.create({
     paddingHorizontal: 10,
     marginHorizontal: 25,
     borderRadius: 12,
-    marginTop: HEIGHT * 0.045,
```

Comment:
I when I use dynamic width/height values, it's usually because I know it can look different of different screen sizes, so I accomodate.

Reply:
yeah, i felt like this one was okay without the dynamic change since it's just a vertically flowing page